Assets/HoloToolkit/Utilities/Editor/SceneLauncherEditor.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.SceneManagement;
+
+namespace HoloToolkit.Unity
+{
+    [CustomEditor(typeof(SceneLauncher))]
+    public class SceneLauncherEditor : Editor
```

Comment:
I'll have to get back to you on this. I've got a few other priority items to take care of.